detectors/node/opentelemetry-resource-detector-github/package.json

```diff
@@ -60,8 +60,8 @@
     "typescript": "4.1.3"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.15.0",
-    "@opentelemetry/core": "^0.15.0",
-    "@opentelemetry/resources": "^0.15.0"
+    "@opentelemetry/api": "^0.16.0",
+    "@opentelemetry/core": "^0.16.0",
```

Comment:
it looks strange to have `"@opentelemetry/core": "^0.16.0"` here and `"@opentelemetry/core": "0.16.0"` in dev dependencies.
is there a reason why we use fixed versions in dev dependencies but not in prod dependencies?

Reply:
it should only depend on resources anyways. good catch

Reply:
devDependencies are pinned so the CI is faster. prod dependencies are ranged so customers can receive important bugfixes without manual intervention.

Reply:
devDependencies are pinned so the CI is faster. prod dependencies are ranged so customers can receive important bugfixes without manual intervention.